spec/support/factories/potions.rb

```diff
@@ -4,7 +4,7 @@
   factory :potion do
     game
 
-    name { 'Deadly Poison' }
+    name { 'My Potion' }
```

Comment:
In general, we want the default name/title for factories of non-canonical models to match the default name/title for the corresponding canonical model. In this case, that name is `'My Potion'`.